Content.Server/Salvage/SalvageSystem.cs

```diff
@@ -155,19 +169,12 @@ private void OnExamined(EntityUid uid, SalvageMagnetComponent component, Examine
                     args.PushMarkup(Loc.GetString("salvage-system-magnet-examined-releasing"));
                     break;
                 case MagnetStateType.CoolingDown:
-                    args.PushMarkup(Loc.GetString("salvage-system-magnet-examined-cooling-down"));
+                    if (_gotGrid == true)
```

Comment:
Same here on style guide, don't use == true.

Reply:
Fixed, thank you